packages/tizen_rpc_port/example/client/lib/message_client.dart

```diff
@@ -135,7 +126,11 @@ class Message extends ProxyBase {
 
     late Parcel parcelReceived;
     while (true) {
-      parcelReceived = _consumeCommand(port);
+      parcelReceived = Parcel.fromPort(port);
+      final int cmd = parcel.readInt32();
+      if (cmd != _MethodId.result.id) {
+        continue;
+      }
```

Comment:
This part has been added to suppress an `unused_field` warning regarding `_MethodId.result`.

Reply:
I forgot this.
I'll add it. Thanks :)